src/main/resources/logback.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<configuration>
+    <appender name="stdout" class="ch.qos.logback.core.ConsoleAppender">
+        <encoder>
+            <pattern>[%d] %p %m \(%c:%t\)%n</pattern>
```

Comment:
This file was created using the (log4j.properties to logback.xml Translator)[http://logback.qos.ch/translator/].

The output is the same we had with log4j in master (see https://github.com/mesosphere/chaos/commit/9ceaa5a6ba823fde19db5d01486739bf58d632f4#diff-df4f6f706036a0c5d315105bff3710adR5):

`[2015-10-16 16:28:41,913] INFO Task task_a has been killed. Waiting for 1 more tasks to be killed. (mesosphere.marathon.upgrade.AppStopActor:System-akka.actor.default-dispatcher-6)`

I agree our format is not the nicest, but I'd change it in a different PR.